src/assets/locales/fr.json

```diff
@@ -363,9 +413,11 @@
   "preferences-screen.preference.prefer-mobile-object-info-panel": "Préférer le panneau mobile d'informations d'objet",
   "preferences-screen.preference.preferred-camera": "Caméra préférée",
   "preferences-screen.preference.preferred-mic": "Micro préféré",
+  "preferences-screen.preference.show-audio-debug-panel": "Montrer le panneau de deboguage audio",
```

Comment:
"preferences-screen.preference.show-audio-debug-panel": "Show Audio Debug Panel",

Reply:
It's actually "débogage" :) Fixed in https://github.com/mozilla/hubs/pull/5002/commits/c83676fe29465d388148063b26aabb0c86651919